coreneuron/CMakeLists.txt

```diff
@@ -59,7 +59,7 @@ include(NmodlHelper)
 # Command for MOD to CPP conversion
 # =============================================================================
 set(CORENRN_NMODL_COMMAND env "MODLUNIT=${NMODL_UNITS_FILE}"
-                              "PYTHONPATH=$ENV{PYTHONPATH}:${CMAKE_BINARY_DIR}/lib/python"
+                              "PYTHONPATH=${CORENRN_NMODL_PYTHONPATH}"
```

Comment:
I thought `$ENV{PYTHONPATH}:` is required when nrnivmodl-core after installation.

Reply:
I checked and the only place where `PYTHONPATH` is set is in
https://github.com/BlueBrain/CoreNeuron/blob/f0daaa460edd3251894ece5a1d8299288cb38ce9/CMakeLists.txt#L204
and 
https://github.com/BlueBrain/CoreNeuron/blob/f0daaa460edd3251894ece5a1d8299288cb38ce9/CMakeLists.txt#L210
Then `CORENRN_NMODL_PYTHONPATH` includes the `PYTHONPATH` set above
https://github.com/BlueBrain/CoreNeuron/blob/f0daaa460edd3251894ece5a1d8299288cb38ce9/CMakeLists.txt#L212
so I thought it's fine.
`mod2c` doesn't need `PYTHONPATH` I believe, so adding an extra `PYTHONPATH` there is not needed, right?